docs-chef-io/content/inspec/resources/parse_config_file.md

```diff
@@ -50,6 +50,13 @@ where each test
 - May run a command to `stdout`, and then run the test against that output
 - May use options to define how configuration data is to be parsed
 
+### Option names containing dots
```

Comment:
```suggestion
### Option Names Containing Periods
```

Website formatting prefers to have the first letter of each word in a header capitalized.